hf_vision_model_tfserving_gke/gcloud_auth.sh

```diff
@@ -0,0 +1,7 @@
+export GCP_PROJECT_ID=gcp-ml-172005
+export SA_KEY_FILE=/tmp/gcp-ml-172005-5c6ccb3685b9.json
+
+echo $SA_KEY_FILE
+
+gcloud auth activate-service-account --key-file $SA_KEY_FILE
+gcloud config set project $GCP_PROJECT_ID
```

Comment:
Instead of this, how about we do `gcloud auth login` and `gcloud config set project $GCP_PROJECT_ID`?

Reply:
sounds good since that removes the dependency of Service Account Key